lib/gateways/planGateway.js

```diff
@@ -1,30 +1,43 @@
 import { nanoid } from 'nanoid';
 import { ArgumentError, Plan } from '../domain';
 
+function toDynamoModel(plan) {
+  return {
+    ...plan,
+    queryFirstName: plan.firstName.toLowerCase(),
```

Comment:
Ben mentioned this morning that some people can have special characters in their name, eg. "François". Could we possible write a normalisation function to do more than just the lowercase?

Reply:
Ahh yeah, sure if we are not certain that names will be consistently entered we could do something to strip out all the combining diacritics characters. Shouldn't be too tricky...

Reply:
Cool, updated now!